verification/cbmc/sources/make_common_data_structures.c

```diff
@@ -30,12 +30,13 @@
 #include <proof_helpers/proof_allocators.h>
 
 void ensure_alg_properties_attempt_allocation(struct aws_cryptosdk_alg_properties *const alg_props) {
+    if (alg_props == NULL) return;
```

Comment:
Other functions in make_common_data_structures and our proof work in c-common use the template of wrapping the function in `if (alg_props != NULL)` instead. Let's use that just to be consistent.

Reply:
Yes, I agree with this. Please check the new style.